src/components/Search/beta/SearchInput.tsx

```diff
@@ -1,24 +1,46 @@
-import type { Dispatch, SetStateAction } from 'react';
+import { Dispatch, SetStateAction, useEffect, useState } from 'react';
 
 import SearchIcon from '@mui/icons-material/Search';
 import FormControl from '@mui/material/FormControl';
 import InputAdornment from '@mui/material/InputAdornment';
 import TextField from '@mui/material/TextField';
-import { connect } from 'react-redux';
 import { style } from 'typestyle';
 
-import type { RootState } from '../../../common/store';
 import useHandleChangeSearch from '../../../hooks/useHandleChangeSearch';
-import { Strings } from '../../../resources/Strings';
 import { InputStylesRaw, Spacing } from '../../../styles';
 
-const strings = Strings.components.searchDefault.base.collapedBase;
-
-function SearchInput(props: SearchInputProps) {
-  const { setFocused, inputError, inputHelperText, view, mode } = props;
+function SearchInput({
+  setFocused,
+  view,
+  mode,
+  inputPlaceholder,
+  base,
+  inputError,
+  inputHelperText,
+}: SearchInputProps) {
   const { handleChangeSearch } = useHandleChangeSearch();
+  //searchType is to distinguish between base and new search inputs for handleChangeSearch hook
+  const [searchState, setState] = useState({
+    baseSearch: '',
+    newSearch: '',
+    searchType: base,
+  });
   const size = view == 'compare-results' ? 'small' : undefined;
 
+  useEffect(() => {
+    handleChangeSearch(searchState);
```

Comment:
It's not clear to me why you don't call it directly in `updateSearchState` below.